src/components/forms/InputCustomField.tsx

```diff
@@ -52,7 +52,14 @@ export function InputCustomField(props: Props) {
       )}
 
       {type === AvailableTypes.Switch && (
-        <Toggle onChange={props.onValueChange} checked={props.defaultValue} />
+        <Toggle
+          onChange={props.onValueChange}
+          checked={
```

Comment:
@beganovich I guess that due to some other cases involving different custom fields, we consistently sent `value.toString()` as the value to custom_fields. However, in this situation, even when we set it to "false," we were consistently receiving a "true" value (checked) instead of "unchecked." So, I have addressed this particular case here.